src/parser/warlock/affliction/modules/features/Checklist/Component.js

```diff
@@ -49,6 +49,7 @@ const AfflictionWarlockChecklist = ({ combatant, castEfficiency, thresholds, sha
         <DotUptime id={SPELLS.SHADOW_EMBRACE.id} thresholds={thresholds.shadowEmbrace} />
         {combatant.hasTalent(SPELLS.SIPHON_LIFE_TALENT.id) && <DotUptime id={SPELLS.SIPHON_LIFE_TALENT.id} thresholds={thresholds.siphonLife} />}
         {combatant.hasTalent(SPELLS.HAUNT_TALENT.id) && <DotUptime id={SPELLS.HAUNT_TALENT.id} thresholds={thresholds.haunt} />}
+        {combatant.hasCovenant(COVENANTS.KYRIAN.id) && <DotUptime spell={SPELLS.SCOURING_TITHE.id} thresholds={thresholds.scouringTithe} />}
```

Comment:
Not displaying correctly, the `<DotUptime spell` should be `<DotUptime id` and that fixes it.